omod/src/main/webapp/uiError.jsp

```diff
@@ -1,12 +1,7 @@
-<%@ include file="/WEB-INF/template/include.jsp"%>
-<%@ include file="/WEB-INF/template/header.jsp"%>
-
 <h1>UI Framework Error</h1>
```

Comment:
Shouldn't this be enough? https://qa-refapp.openmrs.org/openmrs/module/webservices/rest/apiDocs.htm#!/moduleaction/createModuleAction
https://issues.openmrs.org/browse/RESTWS-627